pyprof/nvtx/nvmarker.py

```diff
@@ -43,12 +43,39 @@
 import importlib
 from .config import Config
 from .dlprof import DLProf
+from .dlprof import dprint
 
 # Singleton object tracking dlprof specific information
 dlprof = DLProf()
 # flag to control wrapping ops in nvtx markers
 wrappers_enabled = True
+# List to keep track of callids when there are nested monket patch
+# function calls
+patch_list       = []
```

Comment:
Is there a way to avoid using a global here?

Reply:
moved this to the dlprof class